src/components/App.js.jsx

```diff
@@ -3,8 +3,23 @@ import { Link } from 'react-router'
 import '../css/App.css'
 import BuilderContainer from './containers/BuilderContainer.js.jsx'
 import DashboardContainer from './containers/DashboardContainer.js.jsx'
+import Ajax from './helpers/ajax.js'
+import { connect } from 'react-redux' 
 
 class App extends Component {
+
+  componentDidMount() {
+    const api = new Ajax()
+    api.fetchTags().then(response => this.storeTags(response.data))
```

Comment:
As long as you're using redux, it might not hurt to have an action which takes care of this call. Then the component can be even dumber, by doing

```
import { fetchTags } from '/path/to/actions/or/whatevs'

...

componentDidMount() {
  this.props.dispatch(fetchTags())
}
...
```

That also lets you not import your ajax helper in this component, and the logic in `storeTags` can move out as well.

This is a kind of straightforward idea but actually understanding it and implementing it can be tricky (I had a hard time figuring out best practices architecture) so let me know if I can elaborate!